code/modules/mob/dead/observer/spells.dm

```diff
@@ -42,7 +42,7 @@ GLOBAL_LIST_INIT(boo_phrases, list(
 	if(target.get_spooked())
 		var/area/spook_zone = get_area(target)
 		if (spook_zone.is_haunted == TRUE)
-			to_chat(usr, "<span class='shadowling'>The veil is weak in [spook_zone], it took less effort to influence [target].</span>")
+			to_chat(usr, "<span class='notice'>The veil is weak in [spook_zone], it took less effort to influence [target].</span>")
```

Comment:
instead of removing this span class, would you rename it instead? I used it with dantalion because it fits the spooky theme aesthetic.